src/components/organisms/topbar/topbar.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export const TitleTopbar = ({ title }: { title: string }) => {
+	return (
+		<div className="flex items-center justify-between gap-5 rounded-b-xl bg-gray-1250 py-3 pl-7 pr-3.5">
+			<div className="relative flex items-end gap-3 font-fira-code text-gray-500">
```

Comment:
Don't need this div. move this class to `span` font-fira-code text-gray-500

Reply:
fixed